stories/covid-vax/covid_vax.ipynb

```diff
@@ -0,0 +1,1527 @@
+{
```

Comment:
<p>Two narratives, break the second paragraph after ("the number of doses delivered on each weekday".)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>